core/frontend/src/AccuSnap.ts

```diff
@@ -417,7 +417,7 @@ export class AccuSnap implements Decorator {
     if (!this.errorKey)
       return;
 
-    this.explanation = IModelApp.localization.getLocalizedString(this.errorKey);
+    this.explanation = IModelApp.localization.getLocalizedStringWithNamespace("iModelJs", this.errorKey);
```

Comment:
Do we have to use the `getLocalizedStringWithNamespace` here or would it be equivalent to use,

`IModelApp.localization.getLocalizedStringWithNamespace(`iModelJs:${this.errorKey}`);`

like we do in some of the other methods?

Reply:
Both achieve the same effect. I opted for `withNamespace` here because no string formatting was taking place in this call. I can swap those calls that include `withNamespace` out in favor of inline string formatting if you'd prefer.

Reply:
Ah okay, for consistency mind using the `iModelJs:` syntax?

Reply:
Just fixed this.

Reply:
not seeing it, did you push it?

Reply:
Ah, fixed only one of two. Just fixed / pushed.